composition/fetch_definition.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/tarent/lib-servicediscovery/servicediscovery"
 )
 
+const MAX_PRIORITY int = 4294967295
```

Comment:
4294967295 is the maxvalue of an unsigned 32-bit integer. maybe be more specific and use uint32